docs/source/recipes/plot_11_recipe.py

```diff
@@ -0,0 +1,89 @@
+"""
+Plotting the Warming Stripes
+============================
+
+In this recipe, we will plot the Warming Stripes (Climate Stripes) created by
+Professor Ed Hawkins at NCAS, University of Reading. Here we will use the
+`HadCRUT.5.0.1.0 dataset
+<https://www.metoffice.gov.uk/hadobs/hadcrut5/data/current/download.html>`_ for
```

Comment:
Good idea linking to the source, but I think it would be useful to also indicate which dataset(s) specifically are used, so that people can follow along and there is quite a big listing of various datasets available from that page.